### PR TITLE
[HAL-1356] It is not possible to disable SSO on application security domain in Undertow subsystem in Web Console

### PR DESCRIPTION
**Pull request by honza-kasik**
Added test for https://issues.jboss.org/browse/HAL-1356

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/applicationsecuritydomain/ApplicationSecurityDomainSingleSignOnTestCase.java [hunk 1]
```diff
@@ -145,14 +145,37 @@ public void testEnableSSO() throws Exception {
     }
 
     /**
+     * @tpTestDetails Try to disable Single Sign On for selected security domain with already enabled SSO by clicking on
+     * button present in Web Console configuration.
+     * Validate that SSO was disabled in model.
+     */
+    @Test
+    public void testDisableSSO() throws Exception {
+        final Address keyStoreAddress = securityDomainOperations.createKeyStore();
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/applicationsecuritydomain/ApplicationSecurityDomainSingleSignOnTestCase.java]: Please remove `keyStoreAddress` - it is not used.

**Review by zebrik — COMMENTED** [review]
Just one minor suggestion.

**Comment by honza-kasik** [comment]
Thanks! I removed `keyStoreAddress` and amended changes to previous commit.

**Comment by zebrik** [comment]
Thank you! Merging...

**Pull request merged**